haverhill_311_function/app.py

```diff
@@ -17,8 +17,19 @@ def lambda_handler(event, context):
         Lambda Context runtime methods and attributes
     """
     data = qalert.pull_data_test()
-    qalert_requests: List[db.QAlertRequest] = sanitizer.sanitize(
-        qalert_data=data
-    )
-    with db.QAlertDB() as qalert_db:
-        qalert_db.save_many(requests=qalert_requests)
+    if data != []:
+        latest_date = data[0]["createDate"]
+        qalert_requests: List[db.QAlertRequest] = sanitizer.sanitize(
+            qalert_data=data
+        )
+        with db.QAlertDB() as qalert_db:
+            try:
+                qalert_db.save_many(requests=qalert_requests)
+            except Exception as e:
+                print(e)
```

Comment:
couple reasons i think it would be good to allow the exception to take place:
1. more insight into what went wrong when there is an issue
2. if there is an exception here the whole pipeline (function execution) should be invalidated so should let it crash
let me know what u think, does it make sense?